### PR TITLE
updated ResourceManager from master for the profile 2020-09-01-hybrid 

### PR DESCRIPTION
**Pull request by abbhar**
## Description
Az.Resources ResourceManger folder is from master branch( Commit : 880f04cbda66a9ecd336d4eeed4fe035c0f0428f)
The resources is not changed and it is same as that of 0.10.0-preview version

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
@abbhar The CI fails due to the change of Azure DevOps. We have fixed the issues in master. However, azure stack branch is out of sync for a long time.

File: tools/Az/Az.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.10.0'
+ModuleVersion = '0.11.0'
```
Comment (abbhar) [inline tools/Az/Az.psd1:15]: TODO: need to update once we decide the Az version

File: tools/Az/Az.psd1 [hunk 2]
```diff
@@ -53,18 +53,22 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.0.0'; },
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.0.1'; },
                @{ModuleName = 'Az.Billing'; RequiredVersion = '0.10.0'; }, 
-               @{ModuleName = 'Az.Compute'; RequiredVersion = '0.10.0'; }, 
-               @{ModuleName = 'Az.DataBoxEdge'; RequiredVersion = '1.1.0'; }, 
+               # TODO: need to check for compute.
```
Comment (msJinLei) [inline tools/Az/Az.psd1:58]: Could you check with the compute team and then remove the comment?

File: tools/Az/Az.psd1 [hunk 3]
```diff
@@ -53,18 +53,22 @@ DotNetFrameworkVersion = '4.7.2'
...
+               @{ModuleName = 'Az.Compute'; RequiredVersion = '2.7.0'; }, 
+               #Need to check DataboxEdge or StackEdge
```
Comment (msJinLei) [inline tools/Az/Az.psd1:60]: ditto

File: tools/Az/Az.psd1 [hunk 4]
```diff
@@ -53,18 +53,22 @@ DotNetFrameworkVersion = '4.7.2'
...
+               @{ModuleName = 'Az.DataBoxEdge'; RequiredVersion = '1.1.0'; }
                @{ModuleName = 'Az.Dns'; RequiredVersion = '0.10.0'; }, 
                @{ModuleName = 'Az.EventHub'; RequiredVersion = '1.4.3'; }, 
                @{ModuleName = 'Az.IotHub'; RequiredVersion = '0.10.0'; }, 
                @{ModuleName = 'Az.KeyVault'; RequiredVersion = '0.10.0'; }, 
                @{ModuleName = 'Az.Monitor'; RequiredVersion = '1.6.0'; }, 
-               @{ModuleName = 'Az.Network'; RequiredVersion = '0.10.0'; }, 
-               @{ModuleName = 'Az.Resources'; RequiredVersion = '0.10.0'; }, 
-               @{ModuleName = 'Az.Storage'; RequiredVersion = '0.10.0'; }, 
+               # need to check on Az.Network
```
Comment (msJinLei) [inline tools/Az/Az.psd1:67]: ditto

File: tools/Az/Az.psd1 [hunk 5]
```diff
@@ -53,18 +53,22 @@ DotNetFrameworkVersion = '4.7.2'
...
+               @{ModuleName = 'Az.Network'; RequiredVersion = '1.6.0'; }, 
+               @{ModuleName = 'Az.Resources'; RequiredVersion = '0.11.0'; }, 
+            #    Need to check on storage
```
Comment (msJinLei) [inline tools/Az/Az.psd1:70]: ditto

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/Dns/Dns/Az.Dns.psd1 [hunk 6]
```diff
@@ -116,7 +116,7 @@ PrivateData = @{
         ReleaseNotes = '* Release for Azure Stack 2019-03-01-hybrid profile.'
 
         # Prerelease string of this module
-        Prerelease = 'preview'
+        # Prerelease = 'preview'
```
Comment (bganapa) [inline src/Dns/Dns/Az.Dns.psd1:119]: we should not need preview for any of the modules

File: src/IotHub/IotHub/Az.IotHub.psd1 [hunk 7]
```diff
@@ -132,7 +132,7 @@ PrivateData = @{
         ReleaseNotes = '* Release for Azure Stack 2019-03-01-hybrid profile'
```
Comment (bganapa) [inline src/IotHub/IotHub/Az.IotHub.psd1:132]: we should update teh release notes saying this is for 2020-09-01-hybrid, mentioning the supported api version is a plus

File: tools/Az/Az.psd1 [hunk 8]
```diff
@@ -53,18 +53,22 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.0.0'; },
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '2.0.1'; },
```
Comment (bganapa) [inline tools/Az/Az.psd1:56]: do we have az.resources module in this list?

Comment (abbhar) [reply tools/Az/Az.psd1:56]: yes we have - we are using version 0.11.0 for Az.Resources

**Review by msJinLei — APPROVED** [review]

**Comment by isra-fel** [comment]
@bganapa Hi Bala, is this PR pending on anything or ready to merge?

**Review by bganapa — APPROVED** [review]

**Pull request merged**